sdk/data/aztables/README.md

```diff
@@ -1,151 +1,342 @@
-status codes returned for [REST API][tables_rest] requests.
+## Contributing
 
-For example, if you try to create a table that already exists, a `409` error is returned, indicating "Conflict".
+This project welcomes contributions and suggestions. Most contributions require
+you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution.
+For details, visit [https://cla.microsoft.com](https://cla.microsoft.com).
 
+When you submit a pull request, a CLA-bot will automatically determine whether
+you need to provide a CLA and decorate the PR appropriately (e.g., label,
+comment). Simply follow the instructions provided by the bot. You will only
+need to do this once across all repos using our CLA.
 
-### Setting up console logging
+This project has adopted the
+[Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information, see the
+[Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/)
+or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any
+additional questions or comments.
 
-The simplest way to see the logs is to enable the console logging.
-To create an Azure SDK log listener that outputs messages to console use AzureEventSourceListener.CreateConsoleLogger method.
+### Additional documentation
+For more extensive documentation on Azure Tables, see the [Azure Tables documentation][Tables_product_doc] on docs.microsoft.com.
 
+## Known Issues
+A list of currently known issues relating to Cosmos DB table endpoints can be found [here](https://aka.ms/tablesknownissues).
 
-To learn more about other logging mechanisms see [here][logging].
+## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
-## Next steps
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-Get started with our [Table samples][table_client_samples].
+This project has adopted the [Microsoft Open Source Code of Conduct][msft_oss_coc]. For more information see the [Code of Conduct FAQ][msft_oss_coc_faq] or contact [opencode@microsoft.com][contact_msft_oss] with any additional questions or comments.
 
-## Known Issues
+<!-- LINKS -->
+[source_code]:https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/data/aztables
+[Tables_gopackage]:https://aka.ms/azsdk/go/tables
+[Tables_ref_docs]:https://aka.ms/azsdk/go/tables
```

Comment:
Should this go to a different URL than the package?

Reply:
I plan on updating these two after release when they are published. The first will be the go.pkg.dev page and the second will be the ms docs page.